packages/react-components/react-utilities/src/hooks/useOnScrollOutside.ts

```diff
@@ -9,15 +9,15 @@ import type { UseOnClickOrScrollOutsideOptions } from './useOnClickOutside';
 export const useOnScrollOutside = (options: UseOnClickOrScrollOutsideOptions) => {
   const { refs, callback, element, disabled, contains: containsProp } = options;
 
-  const listener = useEventCallback((ev: MouseEvent | TouchEvent) => {
+  const listener = useEventCallback((ev: Event) => {
     const contains: UseOnClickOrScrollOutsideOptions['contains'] =
       containsProp || ((parent, child) => !!parent?.contains(child));
 
     const target = ev.composedPath()[0] as HTMLElement;
     const isOutside = refs.every(ref => !contains(ref.current || null, target));
 
     if (isOutside && !disabled) {
-      callback(ev);
+      callback(ev as MouseEvent | TouchEvent);
```

Comment:
Until now useOnClickOutside and useOnScrollOutside have used the same event types, if they don't then we should update the types to be accurate instead of resorting to casting here

Reply:
If I update `callback: (ev: MouseEvent | TouchEvent) => void;` to `allback: (ev: Event) => void;`, is this breaking api? If not I can make the change.

Reply:
fixed it :) 🔧